test/tools/libtesteth/TestSuite.cpp

```diff
@@ -232,10 +232,10 @@ void TestSuite::executeTest(string const& _testFolder, fs::path const& _testFile
 	}
 
 	// Test is generated. Now run it and check that there should be no errors
-	if ((Options::get().singleTest && Options::get().singleTestName == testname) || !Options::get().singleTest)
-		cnote << "TEST " << testname << ":";
+    if (Options::get().verbosity > 1)
+        std::cout << "TEST " << testname << ":\n";
```

Comment:
It's only this single place for which you first changed verbosity of all `cnote`? Ok then.